state/unit.go

```diff
@@ -1230,9 +1230,13 @@ func (u *Unit) WaitAgentPresence(timeout time.Duration) (err error) {
 func (u *Unit) SetAgentPresence() (*presence.Pinger, error) {
 	presenceCollection := u.st.getPresenceCollection()
 	recorder := u.st.getPingBatcher()
-	p := presence.NewPinger(presenceCollection, u.st.ModelTag(), u.globalAgentKey(),
+	model, err := u.st.Model()
```

Comment:
Is st.ModelUUID() remaining as a method on State? Probably not? If it were, you'd not need this code to get a model (here and elsewhere). I can't see a card to remove ModelUUID() though. But I'd expect that's on the radar?

Reply:
As discussed, this will be done in a followup.